README.md

```diff
@@ -10,9 +10,12 @@ Franklin Dashboard allows you to check and manage your Franklin Projects.
     ```
     	GITHUB_CLIENT_ID=7047fds785d670dfsd9ab46    - App's Github Client ID
```

Comment:
they might be, but they won't be usable by anyone who doesn't have access to her local environment and will likely be expired by the time anyone could try.